android/app/src/main/java/com/mattermost/rnbeta/MainActivity.java

```diff
@@ -1,17 +1,33 @@
 package com.mattermost.rnbeta;
 
+import android.content.Intent;
+import android.content.res.Configuration;
 import android.os.Bundle;
 import androidx.annotation.Nullable;
+import android.view.KeyEvent;
 
 import com.reactnativenavigation.NavigationActivity;
-import android.view.KeyEvent;
+import org.devio.rn.splashscreen.SplashScreen;
 import com.github.emilioicai.hwkeyboardevent.HWKeyboardEventModule;
 
 public class MainActivity extends NavigationActivity {
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
+        Intent intent = new Intent("onConfigurationChanged");
+        intent.putExtra("newConfig", newConfig);
+        sendBroadcast(intent);
+    }
+
+    private boolean isUIModeNight() {
+        return getResources().getConfiguration().uiMode & Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES;
```

Comment:
@migbot Sorry, I must not have rebuilt the app after I made the changes so I didn't see the build errors! You're getting the white background on android because the mask needs to be applied. I've fixed it in the latest commit, I think. Also, what size should the logo be?

Reply:
Hey @courtneypattison, the logo image should be scaled down to 112x112. And I can get you a couple of new images to use - what's the best way to share those with you? Also, I think it should be white when placed on the dark background. In the end it should look like this:

<table>
<tr>
<td>Light</td>
<td>Dark</td>
</tr>
<tr>
<td><img src="https://user-images.githubusercontent.com/2040554/76334883-18f75680-62ca-11ea-9154-76b6f2e5971f.png"></td>
<td><img src="https://user-images.githubusercontent.com/2040554/76334884-198fed00-62ca-11ea-961c-e262039bbbe2.png"></td>
</tr>
</table>

Hope this answers your questions!